lib/ruboty/adapters/slack_socket_mode.rb

```diff
@@ -82,6 +82,22 @@ def delete_interactive(response_url)
         post_as_json(response_url, params)
       end
 
+      def update_interactive_block(response_url, block)
+        params = {
+          replace_original: "true",
+          blocks: block,
+        }
+        post_as_json(response_url, params)
+      end
+
+      def update_interactive_message(response_url, text)
+        params = {
+          replace_original: "true",
+          text: text,
+        }
+        post_as_json(response_url, params)
+      end
```

Comment:
[suggestion] 改めて見て思ったのですが、 `update_interactive` 系はメソッドを分岐しない形の方が好きかなという気がしましたので提案します。

```suggestion
      def update_interactive(response_url, text:, blocks:)
         if text.nil? && blocks.nil?
          Ruboty.logger.warn("#{self.class.name}: Cannot update message. Wrong number of arguments (expected text or blocks)")
          return
        end
      
        params = { replace_original: "true" }
        params[:text] = text if text
        params[:blocks] = blocks if blocks
        post_as_json(response_url, params)
      end
```

`InteractiveMessage` からは、 `response_url` の有無だけチェックしてあとはこれをそのまま呼ぶだけでよいイメージです。

Reply:
提案通りに適応しました。
一点だけ異なり
```def update_interactive(response_url, text:, blocks:)```のtextとblocksはキーワード引数ではないです。
response_urlはキーワード引数でないので合わせた感じです。
全てをキーワード引数にしてもいいと思いますが、```delete_interactive```と合わせるようにしたいなと考えました。